### PR TITLE
Add linux-audit

### PR DESCRIPTION
**Pull request by sshockwave**
The name `linux-audit` is chosen instead of `libaudit` as in Linux distributions because conda is cross-platform and this is a Linux-only package. It also matches the original repo name.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sshockwave** [comment]
@conda-forge/help-c-cpp please review!

File: recipes/linux-audit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - make
+  host:
+    - openldap
+
+test:
+  commands:
+    - test -f "${PREFIX}"/lib/libaudit.so
+    - auvirt --help
+    - '"${PREFIX}"/sbin/aureport -v'
+    - '"${PREFIX}"/sbin/ausearch -v'
```
Comment (carterbox) [inline recipes/linux-audit/meta.yaml:32]: @conda-forge/core, do we ship packages in $PREFIX/sbin or just put everything into $PREFIX/bin?

File: recipes/linux-audit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - automake
+    - libtool
+    - make
+  host:
+    - openldap
+
+test:
+  commands:
+    - test -f "${PREFIX}"/lib/libaudit.so
+    - auvirt --help
```
Comment (carterbox) [inline recipes/linux-audit/meta.yaml:30]: Please add existence test for headers to indicate that they are also shipped.

**Review by carterbox — CHANGES_REQUESTED** [review]

File: recipes/linux-audit/meta.yaml (hunk 1 above)
Comment (chrisburr) [reply recipes/linux-audit/meta.yaml:32]: We only add `$PREFIX/bin` to `$PATH`.

**Comment by sshockwave** [comment]
@caterbox Now that we decided to go with CDT for `linux-pam`(#21955), I'm hesitating to add this package for the same reason. I have neither the expertise nor the time to review the security concerns. May I close this PR?

**Comment by carterbox** [comment]
@sshockwave, it's your PR. You can close it whenever you want to close it.

**Comment by sshockwave** [comment]
@carterbox Alright. Thanks for your review anyway!

**Pull request closed without merging**